synapse/datamodel.py

```diff
@@ -783,6 +783,10 @@ def _addFormUniv(self, form, name, tdef, info):
 
     def addUnivProp(self, name, tdef, info):
 
+        if not s_grammar.isBaseProp(name):
```

Comment:
These would cause pre-existing declarations to blow up. We should only be checking at the storm API boundary to allow existing ones to be deleted and not crash cortexes.